src/apps/content-editor/src/app/views/ItemCreate/ItemCreate.tsx

```diff
@@ -268,7 +274,12 @@ export const ItemCreate = () => {
   }
 
   return (
-    <WithLoader condition={!loading && item} message="Creating New Item">
+    <WithLoader
+      condition={
+        !loading && item && isSuccessNewModelFields && !isFetchingNewModelFields
```

Comment:
My only concern here is wouldn't this just keep on showing the loading screen if the api call to get the model fields fail - `isSuccessNewModelFields`?